client/src/app/page/navbar/navbar-profile/navbar-profile.component.ts

```diff
@@ -48,17 +49,20 @@ export class NavbarProfileComponent implements OnInit {
     return false;
   }
 
-  currentPage(): boolean {
-    this.menuList.map(item => item.current = item.id === 'my-profile');
+  currentPage(currentRouter): boolean {
```

Comment:
looks like this logic repeats on all pages. You need to find workaround how to do it once.

Reply:
If I put it in the service for nav-items, will it ok?